BetaCameras/OrbbecOpenNI/OrbbecOpenNI.h

```diff
@@ -192,18 +192,19 @@ namespace MetriCam2
 				}
 			}
 
-			property ParamDesc<unsigned int>^ IRExposureDesc
-			{
-				inline ParamDesc<unsigned int> ^get()
-				{
-					ParamDesc<unsigned int> ^res = gcnew ParamDesc<unsigned int>();
-					res->Unit = "";
-					res->Description = "IR exposure";
-					res->ReadableWhen = ParamDesc::ConnectionStates::Connected;
-					res->WritableWhen = ParamDesc::ConnectionStates::Connected;
-					return res;
-				}
-			}
+			// Disabled while the IRExposure getter is not implemented
```

Comment:
Don't commit old, commented code. That's what Git's for. Just deleted it and search Git if it's needed in the future. This way, commented code can remain a red warning flag, asking the question: "Why's this commented?"

Reply:
That code will be needed, once IRExposure.get is implemented.
In that sence it is not old, but was ahead of its time.
Plus, I added a comment why it's disabled.

Reply:
Removed unwanted changes to IRExposure.get